setup.py

```diff
@@ -59,5 +59,7 @@ def get_versions():
         [console_scripts]
         dask-cuda-worker=dask_cuda.cli.dask_cuda_worker:go
         dask-config=dask_cuda.cli.dask_config:go
```

Comment:
What would we like to do with this entry point?

Reply:
I assume you're referring to the `dask-config` one, this is from https://github.com/rapidsai/dask-cuda/pull/1006.

Reply:
Ideally I'd like that functionality to move from `dask-cuda` to `dask-ctl`, especially given that it isn't particularly GPU specific. cc @quasiben 

Alternatively if it is going to stay here I expect it would become `dask cuda config` and we would need to update the click groups accordingly.

Reply:
Yep we were discussing renaming in issue ( https://github.com/rapidsai/dask-cuda/issues/1038 ). Feel free to share thoughts there 🙂

If it will take a while to move this functionality to `dask-ctl`, renaming may be prudent as an intermediate step in any event